src/components/backend-ai-summary-view.ts

```diff
@@ -383,29 +383,31 @@ export default class BackendAISummary extends BackendAIPage {
     return num.toString().replace(regexp, ',');
   }
 
-  _refreshInvitations() {
+  _refreshInvitations(refreshOnly = false) {
     if (!this.activeConnected) {
       return;
     }
     globalThis.backendaiclient.vfolder.invitations().then(res => {
       this.invitations = res.invitations;
-      if (this.active) {
+      if (this.active && !refreshOnly) {
         setTimeout(() => {
           this._refreshInvitations()
         }, 15000);
       }
     });
   }
 
-  _acceptInvitation(invitation: any) {
+  _acceptInvitation(e, invitation: any) {
     if (!this.activeConnected) {
       return;
     }
     globalThis.backendaiclient.vfolder.accept_invitation(invitation.id)
       .then(response => {
+        let panel = e.target.closest('lablup-activity-panel');
```

Comment:
`panel` is `null` since `e.target` is `backend-ai-console`, so raises following error in the front.

Reply:
Fixed.